modules/git/repo_language_stats.go

```diff
@@ -6,13 +6,35 @@ package git
 import (
 	"strings"
 	"unicode"
+
+	"code.gitea.io/gitea/modules/optional"
 )
 
 const (
 	fileSizeLimit int64 = 16 * 1024   // 16 KiB
 	bigFileSize   int64 = 1024 * 1024 // 1 MiB
 )
 
+// true if "set"/"true", false if "unset"/"false", none otherwise
+func linguistToBool(attr map[string]string, name string) optional.Option[bool] {
```

Comment:
Name it `gitAttributeToBool` and move it somewhere to the git module? Or does it only apply to those few attributes?

Reply:
It's already in the git module but I can move it to some helper file.

Reply:
Ah yes, maybe `modules/git/attributes.go`.